packages/web-client/app/components/card-pay/deposit-workflow/confirmation/index.css

```diff
@@ -109,62 +97,49 @@
   margin-top: var(--boxel-sp-xs);
 }
 
-.card-pay-confirmation__transaction-token-icon {
-  flex-shrink: 0;
-  width: 2.5rem;
-  height: 2.5rem;
-  margin-right: var(--boxel-sp-xs);
-}
-
 .card-pay-confirmation__down-arrow {
+  position: absolute;
+  left: calc(50% - 1.25rem);
   width: 1.25rem;
   height: 1.25rem;
-  margin: var(--boxel-sp-xs) 0;
-  transform-origin: 50% 50%;
-  transform: rotate(90deg);
+  margin: var(--boxel-sp-xs);
 }
 
 .card-pay-confirmation__action-taken {
   width: 100%;
   padding: var(--boxel-sp-lg);
   background: var(--boxel-purple-100);
+  border-radius: var(--boxel-border-radius);
+  font: var(--boxel-font-sm);
 
   display: flex;
   align-items: center;
   justify-content: center;
 }
 
 .card-pay-confirmation__action-taken-icon {
-  width: 1.25rem;
-  height: 1.25rem;
   margin-right: var(--boxel-sp-sm);
 }
 
 .card-pay-confirmation__bridge-container {
-  --negative-margin: calc(-1 * var(--boxel-border-radius));
-
-  position:relative;
-  z-index: 1;
-  margin-top: var(--negative-margin);
-  margin-bottom: var(--negative-margin);
-
+  position: relative;
+  z-index: 0;
   width: 100%;
-  padding: var(--boxel-border-radius) calc(var(--boxel-sp-lg) / 2);
+  padding: 0 var(--boxel-sp-lg);
 
   background-color: var(--boxel-light-300);
 }
 
 .card-pay-confirmation__bridge-inner {
   width: 100%;
-  padding: var(--boxel-sp-xl) calc(var(--boxel-sp-lg) / 2);
+  padding: var(--boxel-sp) var(--boxel-sp-sm);
```

Comment:
Changing the padding breaks alignment:

![image](https://user-images.githubusercontent.com/39579264/119096438-2ec3e800-ba46-11eb-9ae9-ad501107dc5e.png)

Reply:
Made a small change here to align it better, but if you want to go back to using the older version let me know. The design seems to have been updated since you worked on this, that's why I made the changes. @Aierie

Reply:
This change seems good! Thanks!